src/main/java/uk/gov/hmcts/reform/pip/data/management/service/PublicationService.java

```diff
@@ -164,6 +164,14 @@ private void applyExistingArtefact(Artefact artefact) {
         return artefacts;
     }
 
+    public Artefact getMetadataByArtefactIdAdmin(UUID artefactId, boolean verification, boolean isAdmin) {
```

Comment:
I'm thinking we could call this the same / overload the other metadata one (getMetadataByArtefactId) rather than the admin on the end, as it's not specific to admin (the admin calculation is done as part of finding the artefact by id)